posts/2021/07/introducing-code-generator-v020.md

```diff
@@ -11,7 +11,7 @@
 .. previewimage: /images/pytorch-ignite/code-generator/code-generator-preview.png
 -->
 
-[![Code Generator v0.2.0 demo](https://raw.githubusercontent.com/pytorch-ignite/code-generator/main/src/assets/code-generator-demo.gif)](https://code-generator.pytorch-ignite.ai/)
+[![ Code Generator v0.2.0 demo](https://raw.githubusercontent.com/pytorch-ignite/code-generator/main/src/assets/code-generator-demo.gif)](https://code-generator.pytorch-ignite.ai/)
```

Comment:
```suggestion
[![Demo for PyTorch-Ignite code generator v0.2.0. With a few clicks it can generate the training scripts for a number of tasks such as vision classification.](https://raw.githubusercontent.com/pytorch-ignite/code-generator/main/src/assets/code-generator-demo.gif)](https://code-generator.pytorch-ignite.ai/)
```